.pre-commit-config.yaml

```diff
@@ -16,9 +16,9 @@ repos:
       - id: python-check-blanket-noqa
```

Comment:
Cant comment higher, but pygrep hook can be removed from pre-commit as ruff covers it already (rule PGH)

Reply:
Thanks! I dropped it in the amended commit. The tests have also been fixed.